cockroachcloud/run-bulk-operations.md

```diff
@@ -108,7 +104,7 @@ Read the [`IMPORT`](../{{site.current_cloud_version}}/import.html) page for more
 
 ### Export data out of {{ site.data.products.db }}
 
-Using `EXPORT` with `userfile` is not recommended. If you need to export data from a {{ site.data.products.serverless }} cluster, you can either [set up billing for your organization](billing-management.html) to access cloud storage or export data to a local CSV file by using [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
+Using `EXPORT` with `userfile` is not recommended. If you want to export data to a local CSV file, you can use [`cockroach sql --execute`](../{{site.current_cloud_version}}/cockroach-sql.html#general). For example:
```

Comment:
The mention of `EXPORT` could be a link to the [`EXPORT` docs](https://www.cockroachlabs.com/docs/stable/export.html) to show examples etc IMO